.github/workflows/e2e.yml

```diff
@@ -3,35 +3,39 @@ name: E2E Testing
 on:
   workflow_dispatch:
   push:
-    branches: [ main ]
+    branches: [main]
     paths-ignore:
-      - 'README.md'
+      - "README.md"
       - .gitignore
       - LICENSE
   pull_request:
-    branches: [ main ]
+    branches: [main]
     paths-ignore:
-      - 'README.md'
+      - "README.md"
       - .gitignore
       - LICENSE
 
 jobs:
   test:
     timeout-minutes: 30
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04 #using a stable version
     steps:
       - name: Checkout project
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Setup Node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: 18
+      - name: Install required dependencies
```

Comment:
This line doesn't seem necessary

Reply:
That's just an extra step to future proof the test. Just incase the package is removed in the current version but we can do without it for now.